apps/meteor/app/message-mark-as-unread/client/actionButton.ts

```diff
@@ -6,29 +6,31 @@ import { messageArgs } from '../../../client/lib/utils/messageArgs';
 import { ChatSubscription } from '../../models/client';
 import { roomCoordinator } from '../../../client/lib/rooms/roomCoordinator';
 import { dispatchToastMessage } from '../../../client/lib/toast';
+import { sdk } from '../../utils/client/lib/SDKClient';
 
 Meteor.startup(() => {
 	MessageAction.addButton({
 		id: 'mark-message-as-unread',
 		icon: 'flag',
 		label: 'Mark_unread',
 		context: ['message', 'message-mobile', 'threads'],
-		action(_, props) {
+		async action(_, props) {
 			const { message = messageArgs(this).msg } = props;
-			return Meteor.call('unreadMessages', message, async function (error: unknown) {
-				if (error) {
+			return sdk
+				.call('unreadMessages', message)
+				.then(async () => {
```

Comment:
since you change `action` to `async`, why are you using `then` instead of `await`?